README.md

```diff
@@ -115,8 +115,43 @@ board.print_connections()  # Print updated connections
 # Disconnect all channels
 board.disconnect_all_channels()
 ```
+### Example 3: Switching from Simulated to I2C-Controller
 
-### Example 3: Resetting the Board
+This example showcases Example 2, but showcases using the I2CDriverBoardController
```

Comment:
```suggestion
This example is similar to Example 2, but demonstrates using the `I2CDriverBoardController` instead of the `SimulatedBoardController`.
```